assets/app/styles/_core.less

```diff
@@ -976,6 +976,16 @@ h1 small.meta, .build-config-summary .meta {
       }
     }
   }
+  .action-inline {
```

Comment:
i don't see anything with the action-inline class being added, is this still needed?